examples/mqtt_rf24ethernet_and_callback_with_argument/mqtt_rf24ethernet_and_callback_with_argument.ino

```diff
@@ -0,0 +1,193 @@
+/*
```

Comment:
This feels like quite a complex example. Saying 'its up to the developer to decide about the design of topic handlers' doesn't give the develop much help to understand how to use this feature.

The example also needs more documentation to explain what its actually doing - it's taken me a while to piece together how it works.